### PR TITLE
HE lore from vanilla LC + You Must Become Strong

### PR DESCRIPTION
**Pull request by RedMageCole**
## About The Pull Request

This PR adds lore for the HE-class Abnormalities originating from the original Lobotomy Corporation. It also contains lore written by myself for You Must Become Strong. There are a couple of tweaks to the code for the ZAYIN stuff just to hopefully make it look nicer.

## Why It's Good For The Game

Reading is fun because it makes you stronger. You want to become strong, right?

## Changelog
:cl:
tweak: Tweaked the formatting of the ZAYIN abnormality data
add: Added Happy Teddy Bear lore
add: Added All-Around Helper lore
add: Added Scarecrow lore
add: Added Rudolta lore
add: Added Der Freischutz lore
add: Added Woodsman lore
add: Added Laetitia lore
add: Added Funeral lore
add: Added Porccubus lore
add: Added Child of the Galaxy lore
add: Added Singing Machine lore
add: Added Schadenfreude lore
add: Added You Must Become Strong lore
add: Added Nameless Fetus lore
/:cl:

### CODE REVIEW

File: code/modules/paperwork/records/lore/he.dm [hunk 1]
```diff
@@ -1,46 +1,451 @@
...
+
+	<h3><Warning>  Only employees Level 3 and higher are permitted to read this document.</h3><br>
+	"(This recording contains a call from the Containment Unit’s emergency phone. Phone systems are installed in every room of Abnormalities of Risk Level HE or higher to enable quick threat response.)<br>
+	Employee: Breach detected! I'm entering the containment unit right now!... What the?<br>
+	???: Ah. Ah. Highly excited!!! / Ahh~ Thoroughly amusing!!! / Da GAME of DEATH! (Three of the same voice can be heard speaking at different times.)<br>
+	Employee: The hell are those things?<br>
+	???: Strong! STRONG!!! / So excited!!! / Worktime! Worktime!<br>
+	Employee: Uh... Okay... Hello, ERD? This is the Containment Unit of 'You Must Become Strong...?'<br>
+	The Emergency Response Department (ERD): Hello, this is the Emergency Response Team. What’s the problem?<br>
```
Comment (Kitsunemitsu) [inline code/modules/paperwork/records/lore/he.dm:376]: I'd replace ERD with Safety or welfare, minor thing

Comment (RedMageCole) [reply code/modules/paperwork/records/lore/he.dm]: All set! ERD was a carryover from Nothing There's lore, but I don't think it appears in any other lore log so I can rewrite it for consistency when I get to porting in NT. Thank you!

**Comment by vampirebat74** [comment]
image
minor writing nitpick, I do this all the time though

**Review by Kitsunemitsu — APPROVED** [review]
Another one down.

**Pull request merged**